packages/tpetra/core/src/Tpetra_Details_unpackCrsGraphAndCombine_def.hpp

```diff
@@ -970,15 +1006,14 @@ unpackCrsGraphAndCombineNew(
   }
   auto imports_d = imports.template view<buffer_memory_space>();
 
-  auto local_graph = sourceGraph.getLocalGraph();
-  auto local_col_map = sourceGraph.getColMap()->getLocalMap();
-  typedef decltype(local_col_map) local_map_type;
-
   // Now do the actual unpack!
-  UnpackAndCombineCrsGraphImpl::unpackAndCombine<
-    packet_type,local_graph_type,local_map_type,buffer_device_type>(
-      local_graph, local_col_map, imports_d, num_packets_per_lid_d,
-      import_lids_d, combineMode, false, atomic);
+  // TJF: Should be grabbed from the Graph
+  indices_type indices;
```

Comment:
Is there something missing here or is it OK?

Reply:
i don't think so.  probably a place holder comment from the first iterations.  I'll kill the comment on the next round of work that will need to be done with the new interface.